### PR TITLE
add period to copyright

### PR DESCRIPTION
**Pull request by zdiaz2826**

### CODE REVIEW

**Comment by zdiaz2826** [comment]
@rv-mkroh

File: 01_Core/week02/submissions/Review/index.html [hunk 1]
```diff
@@ -0,0 +1,86 @@
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <title>My Page</title>
+        <link rel="stylesheet" type="text/css" href="style.css">
+    </head>
+    <body>
+        <div class="sidebar">
```
Comment (rv-mkroh) [inline 01_Core/week02/submissions/Review/index.html]: sidebars typically use the <aside> tag.

File: 01_Core/week02/submissions/Review/index.html [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+                <ul>
+                <li><a href="https://www.google.com">Google</a></li>
+                <li><a href="https://www.foxnews.com">News</a></li>
+                <li><a href="https://weather.com">Weather</a></li>
+            </ul>
+        </div>
+        <div class="topnav">
```
Comment (rv-mkroh) [inline 01_Core/week02/submissions/Review/index.html]: This is a nav so use a <nav> tag and then inside of that use a <ul> for your list of links.

File: 01_Core/week02/submissions/Review/index.html [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+                <li><a href="https://weather.com">Weather</a></li>
+            </ul>
+        </div>
+        <div class="topnav"> 
+                <a href="#">One</a>
+                <a href="#">Two</a>
+                <a href="#">Three</a>
+                <a href="#">Four</a>
+                <div class="dropdown"> 
+                <button class="dropbtn" onclick="myFunction()"> Five
```
Comment (rv-mkroh) [inline 01_Core/week02/submissions/Review/index.html]: Work on your indentation. <button> is a child of the <div> so should be indented.

**Pull request closed without merging**